src/Component/src/Generator/RandomnessGenerator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Generator;
+
+use Webmozart\Assert\Assert;
+
+final class RandomnessGenerator implements RandomnessGeneratorInterface
+{
+    private string $uriSafeAlphabet;
+
+    private string $digits;
+
+    public function __construct()
+    {
+        $this->digits = implode(range(0, 9));
```

Comment:
Nitpick, but would be cool to work with `$this->digits` and `$this->uriSafeAlphabet` in the same order as class properties defined.